### PR TITLE
feature(condo): DOMA-5132 apple push adapter

### PR DESCRIPTION
**Pull request by akarjakin**
- Added Apple push adapter that supports both ordinary and VoIP push notifications
- integrated it into push handler of Message domain
- added/updated tests
- added apple config to .helm

### CODE REVIEW

File: apps/condo/domains/notification/adapters/apple/session.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+const { isEmpty, get } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const { getCurrSec } = require('@condo/domains/common/utils/date')
+
+const SESSION_LIFE_TIME = 60 * 60 * 24
+const SESSION_PING_INTERVAL = 60
+// const APPLE_API_ENDPOINT = 'https://api.sandbox.push.apple.com'
```
Comment (sitozzz) [inline apps/condo/domains/notification/adapters/apple/session.js:11]: Debug line?
```suggestion
```

File: apps/condo/package.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
     "immer": "9.0.13",
     "isomorphic-fetch": "^3.0.0",
     "js-cookie": "^2.2.1",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.0",
```
Comment (sitozzz) [inline apps/condo/package.json:110]: Why it's need to be updated?

File: apps/condo/domains/notification/adapters/appleAdapter.js [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+const logger = getLogger('appleAdapter')
+
+/**
+ * Send push notification to pushToken via app, configured by APPLE_CONFIG in .helm (.env)
+ * Attention! Notifications could only be sent to devices, connected via same PROJECT_ID.
+ * Attempts to send push notifications to devices, connected through different projects will fail.
+ */
+class AppleAdapter {
+    app = null
+    projectId = null
```
Comment (sitozzz) [inline apps/condo/domains/notification/adapters/appleAdapter.js:36]: There are no usages outside class. Maybe better to make it private?

File: apps/condo/domains/notification/adapters/apple/session.js (hunk 1 above)
Comment (akarjakin) [reply apps/condo/domains/notification/adapters/apple/session.js:11]: will be removed later, after production support added on mobile apps.

File: apps/condo/domains/common/utils/date.js [hunk 4]
```diff
@@ -30,9 +30,16 @@ const getStartDates = (dateRaw) => {
     }
 }
 
+/**
+ * Returns current seconds count from Epoch start UTC
+ * @returns {number}
+ */
+const getCurrSec = () => Math.floor(Date.now() / 1000)
```
Comment (sitozzz) [inline apps/condo/domains/common/utils/date.js:37]: It's implementation of timestamp. Maybe better name it something like `getTimestamp`?

File: apps/condo/package.json (hunk 2 above)
Comment (akarjakin) [reply apps/condo/package.json:110]: v8.5.1 is of 2019 year, and there was no version dependant usages in code. In other apps it's version is already updates, so I decided to make it consistent. Also latest version has some security updates.

**Review by sitozzz — APPROVED** [review]

File: apps/condo/domains/notification/adapters/apple/messaging.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+    ERRORS,
+    APS_PUSH_TYPE_ALERT,
+    APS_PUSH_TYPE_VOIP,
+    APS_PUSH_TYPE_BACKGROUND,
+    APS_RESPONSE_STATUS_SUCCESS,
+} = require('./constants')
+const SessionAPNs = require('./session')
+const JSONWebTokenAPNs = require('./token')
+
+const logger = getLogger('AppleMessaging')
```
Comment (pahaz) [inline apps/condo/domains/notification/adapters/apple/messaging.js:18]: use the package name or adapter name

File: apps/condo/domains/notification/adapters/apple/messaging.js [hunk 6]
```diff
@@ -0,0 +1,170 @@
+const { isEmpty, get } = require('lodash')
```
Comment (pahaz) [inline apps/condo/domains/notification/adapters/apple/messaging.js:1]: you need to rename this file to a class name if you use it only for one class

File: apps/condo/domains/notification/adapters/apple/session.js [hunk 7]
```diff
@@ -0,0 +1,113 @@
+const http2 = require('http2')
```
Comment (pahaz) [inline apps/condo/domains/notification/adapters/apple/session.js:1]: you need to rename this filename to class name

File: apps/condo/domains/notification/constants/constants.js [hunk 8]
```diff
@@ -571,6 +576,7 @@ const APP_RESIDENT_KEY = 'resident'
 const APP_RESIDENT_ID_ANDROID = 'ai.doma.client'
 const APP_RESIDENT_ID_IOS = 'ai.doma.clients'
 const APP_MASTER_ID_ANDROID = 'ai.doma.master'
+const APP_MASTER_ID_IOS = 'ai.doma.master'
```
Comment (pahaz) [inline apps/condo/domains/notification/constants/constants.js:579]: it looks like a hardcode! we need to move it to ENV

**Review by pahaz — APPROVED** [review]

**Pull request merged**